net/ampr-ripd/Makefile

```diff
@@ -0,0 +1,97 @@
+include $(TOPDIR)/rules.mk
+
+PKG_NAME:=ampr-ripd
+PKG_VERSION:=2.4
+PKG_RELEASE:=1
+PKG_SOURCE_URL:=https://www.yo2loj.ro/hamprojects
+PKG_HASH:=18d00c898d22a5ee1fb2c153a49358100b5ff22b2b57668472a651998fb0bf6f
+PKG_MAINTAINER:=Dan Srebnick <k2ie@k2ie.net>
+
+PKG_SOURCE:=$(PKG_NAME)-$(PKG_VERSION).tgz
+PKG_LICENSE:=GPL-2.0-only
+
+include $(INCLUDE_DIR)/package.mk
+
+define Package/ampr-ripd
+	SECTION:=net
+	CATEGORY:=Network
+	SUBMENU:=Routing and Redirection
+	DEPENDS:=+kmod-ipip +ip-full
```

Comment:
Do you need `ip-full` or is `ip-tiny` enough?

Reply:
> Do you need `ip-full` or is `ip-tiny` enough?

ip-full is definitely required.

Reply:
Are you sure? By your init script or anything else? Why is not ip-tiny sufficient?

Reply:
I quickly went through the PR and I don't see any `ip` commands that are missing from `ip-tiny`. So which features of `ip-full` are required?